### ISSUE TITLE
Swift support view swift-json-gen

### ISSUE THREAD

**Issue by samskiter** [cknadler/nidyx#37]
JSONModel is Obj-C only. Would you consider support swift via something like Swift-json-gen (https://github.com/tomlokhorst/swift-json-gen) until proper reflection API's come along to allow JSONModel to work with swift?

**Comment by cknadler** [comment]
Yeah, I would. I don't have a ton of experience with Swift, but as long as there are test cases, that sounds great. :+1:

**Comment by samskiter** [comment]
I'm just getting going in Swift but if all I need is to make the struct, we this could be pretty trivial. I'll have a look at your templates a little more

**Comment by sethfri** [comment]
I'm currently working on getting Nidyx to spit out plain Swift models (no JSON serialization or deserialization support at first).

There are a ton of JSON frameworks out there for Swift at this point; we should examine which one(s) we want to support in Nidyx.

**Comment by samskiter** [comment]
@sethfri How are you getting on with this? the serialization/deserialization work looks to be there with swift-json-gen right? so if you had a model being spat out, it would hopefully be a matter of tying these things together to  get a working solution? :)

**Comment by samskiter** [comment]
@sethfri Wonder if you would mind sharing what you had so far?
